CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.20)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
While this, and all other such changes, is fine, file `examples/CMakeLists.txt` should be updated too I think.

Reply:
Somehow I missed that one. Thanks for the hint!